swagger.json

```diff
@@ -1804,6 +1804,42 @@
                 }
             }
         },
+        "/industrial/data/{data_input_id}": {
+            "get": {
+                "tags": [
+                    "Industrial", "Default"
+                ],
+                "summary": "/industrial/data/{data_input_id}",
```

Comment:
In other API endpoints we define bind variables as `{data_input_id:[0-9]+}`

Reply:
done!